internal/deviations/deviations.go

```diff
@@ -156,6 +156,16 @@ func ISISLevelAuthenticationNotRequired(_ *ondatra.DUTDevice) bool {
 	return *isisLevelAuthenticationNotRequired
 }
 
+// IsisSingleTopologyRequired sets isis af ipv6 single topology on the device if value is true.
+func IsisSingleTopologyRequired(_ *ondatra.DUTDevice) bool {
```

Comment:
I know this wasn't the case with the flag, but Go style says this needs to start with ISIS

Reply:
done